### DISCUSSION TITLE
Invalid ROS dockers

### DISCUSSION THREAD

**Discussion by Rick-v-E** [FieldRobotEvent/Virtual_Field_Robot_Event#44] [Ideas]
The ROS docker files seems to be broken since yesterday:

```GPG error: http://packages.ros.org/ros/ubuntu focal InRelease: The following signatures were invalid: EXPKEYSIG F42ED6FBAB17C654 Open Robotics <info@osrfoundation.org>```

For people with the same problem as we: a temporary [fix](https://github.com/osrf/docker_images/issues/535 ) for this issue is to add 

```RUN apt-key adv --keyserver 'hkp://keyserver.ubuntu.com:80' --recv-key C1CF6E31E6BADE8868B172B4F42ED6FBAB17C654```

at the top of the docker file.

**Comment by ThijsRuigrok** [comment]
Tnx Rick, this worked for me. When building the docker image I now get the following warning `Warning: apt-key output should not be parsed (stdout is not a terminal)`. But this not a a real problem.
